StripeCore/StripeCore/Source/Analytics/STPAnalyticEvent.swift

```diff
@@ -77,7 +77,21 @@ import Foundation
 
     // MARK: - Embedded Payment Element init
     case mcInitEmbedded = "mc_embedded_init"
-
+    
+    // MARK: - Embedded Payment Element loading
+    case mcLoadStartedEmbedded = "mc_load_started_embedded"
+    case mcLoadSucceededEmbedded = "mc_load_succeeded_embedded"
+    case mcLoadFailedEmbedded = "mc_load_failed_embedded"
+    
+    // MARK: - Embedded Payment Element confirm
+    case mcConfirmEmbedded = "mc_embedded_confirm"
+    
+    // MARK: - Embedded Payment Element update
+    case mcUpdateStartedEmbedded = "mc_embedded_update_started"
+    case mcUpdateFinishedEmbedded = "mc_embedded_update_finished"
+    case mcUpdateFailedEmbedded = "mc_embedded_update_failed"
+    case mcUpdateCanceledEmbedded = "mc_embedded_update_canceled"
```

Comment:
Let's use a single `mc_embedded_update_finished` and send status = canceled, failed, succeeded. Easier to query in hubble.